public/app/config/env/staging.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+    'use strict';
+
+    define(
+        [
+            'util/container',
+            'services/searchService'
+        ],
+        function (container, searchService) {
+            var simpleApiBaseUrl = 'https://staging-collections.histwest.org.au/service.php/simple',
```

Comment:
I don't know either. Maybe an integration test that pokes at the actual web services as part of build / deployment?